app/views/layouts/application.html.erb

```diff
@@ -19,8 +19,8 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
 
     <% if ENV["GOOGLE_ANALYTICS_ID"] %>
-      <%= javascript_include_tag "https://www.googletagmanager.com/gtag/js?id=#{ENV["GOOGLE_ANALYTICS_ID"]}" %>
       <%= javascript_include_tag "google_analytics", data: { "ga-id" => ENV["GOOGLE_ANALYTICS_ID"] } %>
+      <%= javascript_include_tag "analytics" %>
```

Comment:
Can we flip the names of these two or make it clearer that `analytics` is GA, not some other analytics library. What was the name of the source file?

Reply:
Ok, saw you included it in the description. In that case, can we namespace it in a folder. I agree with Robbie's comment, that this feels like it belongs in `vendor/`, but perhaps it would be good to put it in `vendor/assets/javascripts/google_analytics/analytics.js` or something like that.

Reply:
Prettier also ignores things in `vendor/`.